airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/spec.json

```diff
@@ -28,7 +22,64 @@
           "type": "integer"
         },
         "default": []
+      },
+      "credentials": {
+        "title": "Authorization Method",
+        "type": "object",
+        "oneOf": [
+          {
+            "type": "object",
+            "title": "oAuth2.0",
+            "required": ["client_id", "client_secret", "refresh_token"],
+            "properties": {
+              "credentials": {
```

Comment:
I would change it to auth_method or something like that.  Otherwise json path is "credentials[0]/credentials"  

 "credentials[0]/auth_method/..." - looks better

Reply:
Changed